interfaces/Referral-App/src/app/referral/referral.page.html

```diff
@@ -81,11 +82,31 @@
   </div>
 </ion-content>
 
-<ion-footer>
+<ion-content *ngIf="!isSupportedRegion()">
+  <ion-card *ngIf="!category">
+    <ion-card-header>
+      <ion-card-title>
+        The Netherlands Red Cross provides helpful information for
+        <span
+          *ngFor="let supportedRegion of supportedRegions; first as isFirst; last as isLast"
+          >{{ isFirst ? '' : (isLast ? ' and' : ',')}}
```

Comment:
Fancy. :)

We'll need to rework this in a more "scalable" way when we want to make the interface multi-lingual though...